tests/kernel/run_ltp.pm

```diff
@@ -262,6 +262,19 @@ sub run {
     script_run('$LTPROOT/ver_linux');
 
     if ($is_network) {
+        # poo#18762: Sometimes there is physical NIC which is not configured.
+        # One of the reasons can be renaming by udev rule in
+        # /etc/udev/rules.d/70-persistent-net.rules. This breaks some tests
+        # (even net namespaces based ones).
+        # Workaround: configure physical NIS (if needed) and "flush" network
+        # settings with upload_logs().
+        script_run(
+            q( dir=/sys/class/net; ifaces="`basename -a $dir/* | grep -v -e ^lo -e ^tun -e ^virbr -e ^vnet`"; for iface in $ifaces; do config=/etc/sysconfig/network/ifcfg-$iface; if [ `cat $dir/$iface/operstate` = "down" ] && [ ! -e $config ]; then echo "WARNING: create config '$config'"; printf "BOOTPROTO='dhcp'\nSTARTMODE='auto'\nDHCLIENT_SET_DEFAULT_ROUTE='yes'\n" > $config; systemctl restart network; fi; done)
+        );
+        my $tmp = '/root/workaround-empty-file.txt';
+        script_run("touch $tmp");
+        upload_logs($tmp); # deliberatelly send after setting net
+
```

Comment:
@richiejp This is a terrible hack, but that's the only thing I found to fix failures on SLE. What do you think?